src/css/custom.css

```diff
@@ -118,7 +118,8 @@ h1{
   background: #2CAA5F;
 }
 
-.alert--tip{
+/* a.k.a as Tip admonition*/
```

Comment:
?

Reply:
Well in here it is called "success" but in Docusaurus it is called "Tip" so just a note :-)